ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh

```diff
@@ -136,6 +139,7 @@ function set_users () {
 
 
     cat <<EOF >>"${runtime_env}"
+    set +x
```

Comment:
Line 137 has `set +x`. Line 142 may be not needed.

Reply:
PR is LGTM, I can /approve. Would you like to have this minor update?

Reply:
Updated, Thanks